src/main.cpp

```diff
@@ -1361,10 +1361,6 @@ static int main0(int argc, char **argv) {
             fprintf(stderr, "unable to find native libc file: %s\n", err_str(err));
             return main_exit(root_progress_node, EXIT_FAILURE);
         }
-        if ((err = stage2_libc_render(&libc, stdout))) {
-            fprintf(stderr, "unable to print libc file: %s\n", err_str(err));
-            return main_exit(root_progress_node, EXIT_FAILURE);
-        }
```

Comment:
This part should stay. This is when you call `zig libc` and it prints the libc file, which you would then edit (or not) and pass to `--libc`.

Reply:
Updated my PR